deeplake/client/client.py

```diff
@@ -62,13 +64,21 @@ def __init__(self, token: Optional[str] = None):
             remove_token()
             self.token = token or self.get_token()
             self.auth_header = f"Bearer {self.token}"
+        if self._token_from_env:
+            username_reporting = self.get_user_profile()["name"]
```

Comment:
Will the API call be run in every function that initializes the Client? 
Should not be much overhead, just wanted to making sure.

Reply:
@ProgerDav I think it needs to be. There's no good way to solve the problem otherwise, because there's no triggerable event when the env variable is set.